sdk/modelsrepository/azure-iot-modelsrepository/src/test/java/com/azure/iot/modelsrepository/ModelRepositoryIntegrationTests.java

```diff
@@ -30,7 +30,7 @@ public void getModelsSingleDtmiNoDependencies(HttpClient httpClient, ModelsRepos
 
     @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
     @MethodSource("com.azure.iot.modelsrepository.TestHelper#getTestParameters")
-    public void getModelsSingleDtmiWithDeps(HttpClient httpClient, ModelsRepositoryServiceVersion serviceVersion) throws URISyntaxException {
+    public void getModelsSingleDtmiWithDependencies(HttpClient httpClient, ModelsRepositoryServiceVersion serviceVersion) throws URISyntaxException {
```

Comment:
Why is models plural but then followed by "singular". That is confusing to me.

Reply:
Oh I get it. This is a test case, you are calling getModels, but passing it a single dtmi.